docs/README.md

```diff
@@ -0,0 +1,85 @@
+# Autonomy Research Testbed Documentation
+
+These docs are meant to be a _succinct_ reference to commands, packages, and any other
+information that may be useful to document as it relates to the
+`autonomy-research-testbed` platform.
+
+## Table of Contents
+
+1. Design
+    1. [Repository Structure](./design/repository_structure.md)
+    2. [`atk.yml`](./design/atk.md)
+    3. [Dockerfiles](./design/dockerfiles.md)
+    4. [ROS Workspace](./design/ros_workspace.md)
+    5. [Launch System](./design/launch_system.md)
+2. Usage
+    1. [Development Workflow](./usage/development_workflow.md)
+    2. [How to Run](./usage/how_to_run.md)
+3. [Frequently Asked Questions](./misc/faq.md)
+
+## Quick Start
+
+This section provides the main commands necessary to launch various components. Please ensure you understand the previous topics in the [Table of Contents](#table-of-contents) before continuing.
+
+### Install dependencies
+
+Python dependencies are listed in the `requirements.txt` file and can be installed with the following command:
+
+```bash
+pip install -r requirements.txt
+```
+
+In addition, you will need to install docker and docker compose. Please refer to the [official documentation](https://www.docker.com/get-started/) for installation details.
+
+### Start up vnc
+
+You'll probably want to visualize GUI windows, so start up vnc first. The first time around, the image will need to be build so this may take a little while.
+
+```bash
+$ atk dev -u -s vnc
+WARNING  | logger.set_verbosity :: Verbosity has been set to WARNING
+[+] Running 1/1
+ ✔ Container art-vnc  Started
+```
+
+If you see the following warning, **ignore it**. This simply says that substituting the `$DISPLAY` in the `atk.yml` file fails because `$DISPLAY` is unset which is expected. By passing `vnc` as an optional later, this will override the variable.
+```bash
+WARN[0000] The "DISPLAY" variable is not set. Defaulting to a blank string.
+```
+
+> [!NOTE]
+> You can also use x11 if you're _not_ ssh'd to the host. Replace all `vnc` flags in the `--optionals` with `x11` to do this.
+
+### Launch the simulation
+
+The first time you start up the chrono service, it will need to build the image. This may take a while.
+
```

Comment:
State somewhere here (as in highest level of docs) that Optix needs to be downloaded / put in sim folder